OpenEdXMobile/src/main/java/org/edx/mobile/player/PlayerController.java

```diff
@@ -920,22 +921,29 @@ public void playPrevious() {
         }
     }
 
-    public void setSettingsBtnDrawable(boolean isSettingEnabled){
+    public void setSettingsBtnDrawable(boolean isSettingEnabled) {
         if (mSettingsButton != null) {
             if (isSettingEnabled) {
-                mSettingsButton.setIconColor(getResources().getColor(R.color.accentAColor));
+                mSettingsButton.setColorFilter(
+                        ContextCompat.getColor(mSettingsButton.getContext(), R.color.accentAColor),

```

Comment:
Can we move this to UiUtils as well?

Reply:
There is only one use case for this so far. I think we can leave this for now.